src/test/java/com/elasticbox/jenkins/tests/PullRequestLifecycleManagementTest.java

```diff
@@ -122,7 +131,15 @@ public boolean satisfied() {
         waitForDeletion(instances, TimeUnit.MINUTES.toSeconds(10));
         Assert.assertTrue("Deployed instances are not deleted after 10 minutes since the pull request is closed", instances.isEmpty());
 
-        abortBuildOfClosePullRequest();
+        LOGGER.fine("Checking that the running job is aborted");
+        abortBuildOfClosePullRequest(true);
+
+        LOGGER.fine("Checking that the queued job is aborted");
+        jenkins.getInstance().setNumExecutors(1);
```

Comment:
Each test has it's own JenkinsRule object, which is an embedded Jenkins for tests.